grib/src/main/java/ucar/nc2/grib/collection/Grib2Iosp.java

```diff
@@ -429,13 +429,11 @@ protected void addVariableAttributes(Variable v, GribCollectionImmutable.Variabl
     Grib2Customizer.Parameter entry = cust.getParameter(vindex.getDiscipline(), vindex.getCategory(), vindex.getParameter());
     if (entry != null) v.addAttribute(new Attribute("Grib2_Parameter_Name", entry.getName()));
 
-    if (vindex.getLevelType() != GribNumbers.MISSING) {
-      String levelTypeName = cust.getLevelName(vindex.getLevelType());
-      if (levelTypeName != null)
-        v.addAttribute(new Attribute("Grib2_Level_Type", levelTypeName));
-      else
-        v.addAttribute(new Attribute("Grib2_Level_Type", vindex.getLevelType()));
-    }
+    if (vindex.getLevelType() != GribNumbers.MISSING)
+      v.addAttribute(new Attribute("Grib2_Level_Type", vindex.getLevelType()));
+    String ldesc = cust.getLevelName(vindex.getLevelType());
```

Comment:
Nevermind, it looks like that method doesn't exist on `Grib2Customizer` like it does on `Grib1Customizer`. Weird.